### DISCUSSION TITLE
Dynamic Form, lookup com eventos

### DISCUSSION THREAD

**Discussion by DouglasHeydt** [po-ui/po-angular#1068] [Tire suas dúvidas]
Saudações, estou construindo um formulário utilizando o Dynamic Form, um dos campos retornados possui um serviço de dados e portanto é exibido como um lookup. O problema que estou passando é que a partir da linha selecionada no lookup, preciso pegar todos os valores da linha para retornar pra vários campos, para um lookup estático bastaria adicionar o evento (p-selected). 

Como faço isso  na definição do field do Dynamic Form ?

**Comment by jhonyeduardo** [comment]
Boa tarde @DouglasHeydt 

Atualmente não tem essa opção no Dynamic Form, teríamos que analisar como fazer isto, talvez quando for lookup enviar como segundo parâmetro o objeto completo.

**Reply by DouglasHeydt** [reply]
Ok, conseguimos um workaround usando o validate. De qualquer forma, obrigado pela resposta.

**Reply by wsteixeira** [reply]
Bom dia.

Também gostaria de ter acesso aos demais valores, ou ao menos aquele que aparece na tela 'fieldLabel'.

**Reply by emersonbn** [reply]
@DouglasHeydt , você teria um exemplo de como conseguiu essa solução? estou querendo fazer uma coisa semelhante. Mas tentei usar o "validate" do fields<PoDynamicFormField>, mas esta dando erro de validação.
